gradle.properties

```diff
@@ -11,4 +11,5 @@ org.gradle.logging.level=INFO
 wiremockVersion=2.33.2
 freemarkerVersion=2.3.32
 quarkusPoiVersion=1.0.2
+graalVMVersion=22.3.0
```

Comment:
Just a thought;
most up-to-date GraalVM version seems to be 22.3.2.
This is defined as 22.3.0 and the GCB trigger building this is running version 22.3.1. Should these both use same version to ensure compatibilities?